singleheader/CMakeLists.txt

```diff
@@ -17,8 +16,10 @@ set(SINGLEHEADER_REPO_FILES
 )
 set_source_files_properties(${SINGLEHEADER_FILES} PROPERTIES GENERATED TRUE)
 
-if (MSVC)
-  # MSVC doesn't have bash, so we use existing amalgamated files instead of generating them ...
+find_program(BASH bash)
+
+if (NOT BASH)
```

Comment:
I know you're not done with this, but writing because I noticed anyway: I'd probably flip the if and else around to avoid double negative confusion :)

Reply:
Good idea.